packages/mobile/src/components/premium-track-purchase-drawer/StripePurchaseConfirmationButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react'
+
+import { Button } from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { createStripeSession } from 'app/services/buyAudio'
+import { useThemeColors } from 'app/utils/theme'
+
+const messages = {
+  buy: (price: string) => `Buy $${price}`
+}
+
+type StripePurchaseConfirmationButtonProps = {
+  price: string
+}
+
+export const StripePurchaseConfirmationButton = ({
+  price
+}: StripePurchaseConfirmationButtonProps) => {
+  const navigation = useNavigation()
+  const { specialLightGreen1 } = useThemeColors()
+
+  const handleBuyPress = useCallback(async () => {
+    try {
+      const res = await createStripeSession({
+        amount: price,
+        destinationWallet: 'FgMM3AG1qfpEEdYUiw4Zk9ouqsTkheC3rsu4RjvKtyEB'
```

Comment:
See if we can't derive this using `solanaWeb3Manager` - latest libs should have it updated

Reply:
oops my b lol